Apps/Sandcastle/gallery/Clustering.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Cluster labels, billboards and points.">
+    <meta name="cesium-sandcastle-labels" content="Tutorials,Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <table>
+        <tbody><tr>
+            <td>pixel range</td>
+            <td>
+                <input type="range" min="1" max="200" step="1" data-bind="value: pixelRange, valueUpdate: 'input'">
+                <input type="text" size="2" data-bind="value: pixelRange">
+            </td>
+        </tr>
+        <tr>
+            <td>minimum cluster size</td>
+            <td>
+                <input type="range" min="2" max="20" step="1" data-bind="value: minimumClusterSize, valueUpdate: 'input'">
+                <input type="text" size="2" data-bind="value: minimumClusterSize">
+            </td>
+        </tr>
+        </tbody>
+    </table>
+    <div id="toggleClustering"></div>
```

Comment:
Can we make these checkboxes, the toggle button is confusing because you don't know what state it's in.